airbyte-integrations/connectors/source-mailgun/source_mailgun/spec.json

```diff
@@ -0,0 +1,25 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/mailgun",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Source Mailgun Spec",
+    "type": "object",
+    "required": ["private_key"],
+    "additionalProperties": true,
+    "properties": {
+      "private_key": {
+        "type": "string",
+        "description": "Private API key for authenticate with every request",
+        "airbyte_secret": true
+      },
+      "start_timestamp": {
+        "type": "number",
+        "description": "Initial timestamp to gather data from. If omitted, default timeshift (3 days before now ) will be used."
```

Comment:
3 days were choosen as it's the period od storing messages (and related events on the server). So, actually we never need a biggest period for this connector. But technically still provide such functionality. At least because that server's behaviour could be changed the future.